templates/run-contract-setup.sh

```diff
@@ -127,28 +126,6 @@ jq --slurpfile c combined.json '.L1Config.polygonRollupManagerAddress = $c[0].po
 jq --slurpfile c combined.json '.L1Config.polTokenAddress = $c[0].polTokenAddress' genesis.json > g.json; mv g.json genesis.json
 jq --slurpfile c combined.json '.L1Config.polygonZkEVMAddress = $c[0].rollupAddress' genesis.json > g.json; mv g.json genesis.json
 
-# note this particular setting is different for the bridge service!!
```

Comment:
These configuration changes have been removed from the contract installation script and reconciled with the bridge, agglayer and dac deployments in `main.star`. They are retrieved using `combined.json` (and `jq`) before being injected into the template.